modules/installing-rhv-requirements.adoc

```diff
@@ -31,7 +31,7 @@ If you increase the number of virtual machines in the {rh-virtualization} enviro
 * In production environments, each virtual machine must have 120 GiB or more. Therefore, the storage domain must provide 840 GiB or more for the default {product-title} cluster. In resource-constrained or non-production environments, each virtual machine must have 32 GiB or more, so the storage domain must have 230 GiB or more for the default {product-title} cluster.
 * To download images from the Red Hat Ecosystem Catalog during installation and update procedures, the {rh-virtualization} cluster must have access to an internet connection. The Telemetry service also needs an internet connection to simplify the subscription and entitlement process.
 // TBD - What about the disconnected installation alternative?
-* The {rh-virtualization} cluster must have a virtual network with access to the REST API on the {rh-virtualization} Manager.
+* The {rh-virtualization} cluster must have a virtual network with access to the REST API on the {rh-virtualization} Manager. Ensure that DHCP is enabled on this network, because the VMs that the installer creates obtain their IP address using DHCP.
```

Comment:
```suggestion
* The {rh-virtualization} cluster must have a virtual network with access to the REST API on the {rh-virtualization} Manager. Ensure that DHCP is enabled on this network because the VMs that the installer creates obtain their IP address by using DHCP.
```

> ! using noun
To avoid ambiguity, replace this gerund with either "by using" or "that uses".

Reply:
Fixed.